website/cue/reference.cue

```diff
@@ -464,7 +467,21 @@ _values: {
 		]
 	}
 
-	syntax: "file_system_path" | "field_path" | "literal" | "template" | "regex" | "remap_boolean_expression" | "remap_program" | "strftime"
+	syntax: #StringSyntax
+}
+
+#TypeCondition: {
+	required: bool
+
+	#ConditionSyntax: "datadog_search" | "vrl_boolean_expression"
```

Comment:
```suggestion
	#ConditionSyntax: "datadog_search" | "vrl"
```

The `type` is actually just `vrl` in the config loading code:

https://github.com/vectordotdev/vector/blob/e10294c7b41b7328f708c03a37740c9e617270d8/src/conditions/vrl.rs#L23